src/app/models/app.model.ts

```diff
@@ -0,0 +1,34 @@
+export interface Cards {
+  type: string,
+  background_picture_path: string,
+  background_color: string
```

Comment:
в js/ts используется camelCase, т.е. переменная/свойство начинается с маленькой буквы, а каждое следующее слово с большой буквы слитно
```suggestion
  type: string,
  backgroundPicturePath: string,
  backgroundColor: string
```